apps/wpcom-block-editor/src/wpcom/features/live-preview/utils.ts

```diff
@@ -17,7 +17,10 @@ export const getUnlock = () => {
 	let unlock: ( object: any ) => any | undefined;
 	try {
 		unlock = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
-			'I know using unstable features means my theme or plugin will inevitably break in the next version of WordPress.',
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			( window as any ).wpcomGutenberg?.pluginVersion?.startsWith( 'v18.7' )
```

Comment:
Thank you for addressing this, @sejas!

I believe we should support `=> v18.7` as well (e.g., v18.8, v19.x, etc).

Reply:
Yes!, my plan is to remove the old string once I've deployed!
I've created this issue to track it!
https://github.com/Automattic/dotcom-forge/issues/8118

Reply:
This script also runs on Atomic, so I thought we should continue supporting the old string 🥲 
(Sorry if I misunderstand something)

Reply:
After investigating, I've found that `window.wpcomGutenberg.pluginVersion` is not accessible in Atomci sites.

<img width="473" alt="Screenshot 2024-07-05 at 12 09 50" src="https://github.com/Automattic/wp-calypso/assets/779993/4aaf0f8e-46f7-435d-b62f-e7d530f16e47">


I also think users cannot install other Gutenberg versions since it's a managed plugin. In the same way, users cannot use a different WP version.


<img width="966" alt="Screenshot 2024-07-05 at 12 13 54" src="https://github.com/Automattic/wp-calypso/assets/779993/736edd71-f3c4-40ae-ab79-28e4a605e6f1">

**What's true is that users can deactivate the Gutenberg plugin, and then the unlock would need the previous string version.**

One last thought is that we don't support the previous string version `I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.`